Content.Server/Mobs/Mind.cs

```diff
@@ -272,8 +275,15 @@ public void TransferTo(IEntity? entity)
             OwnedComponent = component;
             OwnedComponent?.InternalAssignMind(this);
 
-            if (VisitingEntity?.HasComponent<GhostComponent>() == false)
+            GhostComponent? ghostComponent = null;
+            VisitingEntity?.TryGetComponent(out ghostComponent);
+
+            if (ghostComponent == null || // visiting entity is not a Ghost
+                (ghostComponent != null && !ghostComponent.CanReturnToBody) || // it is a ghost, but cannot return to body anyway, so it's okay
+                ghostCheckOverride) // to force mind transfer, for example from ControlMobVerb
+            {
                 VisitingEntity = null;
```

Comment:
Replacing this with Unvisit() might fix that other issue.

Reply:
it made the screen go black, and then server complained about a missing ActorComponent

Reply:
Something for a different PR then.